src/Forms/src/Xamarin.Forms.Core/HandlerImpl/WebView.Impl.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Xamarin.Platform;
+
+namespace Xamarin.Forms
+{
+	public partial class WebView : IWebView
+	{
+		WebViewSource2 IWebView.Source { get; set; }
```

Comment:
`WebViewSource2 ` must be temporary. Where we use WebViewSource in Xamarin.Forms require BindableProperties.
How are we going to manage these cases?